x-pack/test/common/services/security_solution/roles_users_utils.ts

```diff
@@ -5,32 +5,15 @@
  * 2.0.
  */
 
-import { assertUnreachable } from '@kbn/security-solution-plugin/common/utility_types';
-import {
-  t1AnalystUser,
-  t2AnalystUser,
-  hunterUser,
-  hunterNoActionsUser,
-  ruleAuthorUser,
-  socManagerUser,
-  platformEngineerUser,
-  detectionsAdminUser,
-  readerUser,
-  t1AnalystRole,
-  t2AnalystRole,
-  hunterRole,
-  hunterNoActionsRole,
-  ruleAuthorRole,
-  socManagerRole,
-  platformEngineerRole,
-  detectionsAdminRole,
-  readerRole,
-} from '@kbn/security-solution-plugin/server/lib/detection_engine/scripts/roles_users';
-
-import { ROLES } from '@kbn/security-solution-plugin/common/test';
+import serverlessRoleDefinitions from '@kbn/es/src/serverless_resources/security_roles.json';
+import essRoleDefinitions from '@kbn/security-solution-plugin/common/test/ess_roles.json';
+import { SecurityRoleName } from '@kbn/security-solution-plugin/common/test';
```

Comment:
import as `type SecurityRoleName`. Same in all other imports for `SecurityRoleName`

Reply:
Interesting `linter` didn't catch it.